### ISSUE TITLE
Codegeneration error with subclass that does not add extra properties

### ISSUE THREAD

**Issue by ddjoh** [actonlang/acton#1598]
### Acton Version

0.18.3.20231207.16.30.53

### Steps to Reproduce and Observed Behavior

```
class Aa(object):
    @property
    a: int

    def __init__(self, a):
        self.a = a

    def f(self):
        return self.a

class Bb(Aa):
    def __init__(self, a):
        Aa.__init__(self, a)

    def f(self):
        return self.a * 2

actor main(env):
    print(Aa(1).f())
    print(Bb(1).f())
    env.exit(0)
```

### Expected Behavior

Succesfull compilation

**Comment by plajjan** [comment]
Fixed by #1599. Closing.

**Comment by ddjoh** [comment]
Still fails to compile with acton 0.20.1.20240227.16.11.52
